tests/ingest_agents.py

```diff
@@ -168,6 +168,12 @@ def get_biomaterials(self):
         def get_bundle_manifests(self):
             return self._get_entity_list('bundleManifests')
 
+        def get_submission_manifest(self):
+            return self._get_entity('submissionManifest')
```

Comment:
This one and line 175 will throw an exception when `self._get_entity` returns `None`,

Reply:
why? it's a 'submissionManifest' and 'summary' are parameters to _get_entity

Reply:
Ok, I misread the `(` to a `[`. Sorry. Should have gone to Specsavers. :-D